From 2016590502558caa98c21abad2ce83c5f4804a98 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Mon, 1 Jun 2020 00:20:26 +0200
Subject: [PATCH] doc/man: Fix installation directory

On Unix-like operating systems man pages are commonly installed to
`/usr/share/man` not `/usr/share/stlink/man`.
---
 doc/man/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/man/CMakeLists.txt b/doc/man/CMakeLists.txt
index 23da9e1..9b3c507 100644
--- a/doc/man/CMakeLists.txt
+++ b/doc/man/CMakeLists.txt
@@ -30,7 +30,7 @@ foreach (manpage ${MANPAGES})
     endif ()
 
     if (f AND NOT WIN32)
-        install(FILES ${f} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/man/man1)
+        install(FILES ${f} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/man/man1)
         unset(f)
     endif ()
 endforeach ()
